components/comments.jsx

```diff
@@ -12,11 +75,67 @@ const Comments = ({article}) => {
       </div>
       <p className="article-body">{article.body}</p>
       <div className="single-article-line">
-        <p className="article-comment-count">
-          Comment Count: {article.comment_count}
+        <p className="article-comment-count">Comment Count: {commentCount}</p>
+      </div>
+      <button
+        className="btn"
+        onClick={() => {
+          handleClickPostComment();
+        }}
+        id="post-comment-button"
+      >
+        {isPostCommentShown ? "Hide" : "Post a comment"}
+      </button>
+      <div
+        className={`post-comment-${isPostCommentShown ? "shown" : "hidden"}`}
+      >
+        <p className="empty-comment-msg">
+          {isEmpty ? "Please fill in Username & Comment" : ""}
```

Comment:
This text doesn't show, when trying to post an empty comment I just get a 'something went wrong' message.